Exercise.md

```diff
@@ -0,0 +1,5 @@
+# New-Repo-Rocio
+what app you want to use
+CMS > Usuario / Password
```

Comment:
if you have to choose your program... is it clear which options do you have? where do you ask the user for Username / Password? what will you do with the inputs? store them or throw them out?